src/components/map/Map.tsx

```diff
@@ -55,25 +60,44 @@ export const Map = (props: MapProps) => {
     [currentCoordinatesRef, initialLocation],
   );
 
-  const {mapLines, selectedCoordinates, onMapClick, selectedFeature} =
-    useMapSelectionChangeEffect(
-      props,
-      mapViewRef,
-      mapCameraRef,
-      startingCoordinates,
-    );
+  const {
+    mapLines,
+    selectedCoordinates,
+    onMapClick,
+    selectedFeature,
+    onReportParkingViolation,
+  } = useMapSelectionChangeEffect(
+    props,
+    mapViewRef,
+    mapCameraRef,
+    startingCoordinates,
+  );
+
+  const {bottomSheetCurrentlyAutoSelected} = useMapState();
+
+  const aVehicleIsAutoSelected =
+    bottomSheetCurrentlyAutoSelected?.type ===
+      AutoSelectableBottomSheetType.Scooter ||
+    bottomSheetCurrentlyAutoSelected?.type ===
+      AutoSelectableBottomSheetType.Bicycle;
+
+  const selectedFeatureIsAVehicle =
+    isScooter(selectedFeature) || isBicycle(selectedFeature);
 
   const [geofencingZonesEnabled, geofencingZonesEnabledDebugOverrideReady] =
     useGeofencingZonesEnabled();
 
   const showGeofencingZones =
     geofencingZonesEnabled &&
     geofencingZonesEnabledDebugOverrideReady &&
-    (isScooter(selectedFeature) || isBicycle(selectedFeature));
+    (selectedFeatureIsAVehicle || aVehicleIsAutoSelected);
```

Comment:
Having both a selected feature and auto selected feature in state is not great as it produces duplicate logic and state management.
To unify them requires a deeper refactor, which we need to come back to.